### PR TITLE
DOC: Num causal

### PR DESCRIPTION
**Pull request by daikitag**
Documentation to reflect the non-dependency of the number of causal sites.

### CODE REVIEW

**Comment by jeromekelleher** [comment]
I see... So how would you go about normalising like Martin et al? I think we need to list out the different ways you'd like to normalise here, and show the user how to do it.

File: docs/genetic.md [hunk 1]
```diff
@@ -338,6 +338,90 @@ size output of {py:func}`sim_trait`:
 
 :::{note}
 The effect size output of {py:func}`sim_genetic` is used in simulating
-the genetic values, and not the `trait_df` input in the frequency dependent
-model.
+the genetic values, as it represents the effect sizes after the frequency
+dependence architecture is accounted.
 :::
+
+## Example: Infinitesimal Model Simulation
```
Comment (gregorgorjanc) [inline docs/genetic.md:345]: I advise agains using the term `infinitesimal`. This term was initially used by Fisher in his 1918/1919 paper where he motivated Gaussian distribution by saying that there are infinitely many causal loci with infinitely small effects, hence the `infinitesimal` model. Then for decades people were just simulating quantitative genetic studies with only Gaussian distribution without reference to genome. And for many, this is what `infinitesimal` simulation refers to. 1000 loci is of course a lot and resulting distributions look very much Gaussian, but its not `infinitesimal` as Fisher has coined it. Just my 2 pennies.

Comment (daikitag) [reply docs/genetic.md:345]: Thanks a lot @gregorgorjanc !

File: docs/genetic.md [hunk 2]
```diff
@@ -338,6 +338,90 @@ size output of {py:func}`sim_trait`:
...
+```
+
+We see that the simulated genetic values follow an asymptotic normal distribution. However,
+we see that the variance of the simulated traits will be a very large number, as shown
+in the below output.
+
+```{code-cell}
+
+import numpy as np
+print(np.var(genetic_result.genetic["genetic_value"]))
```
Comment (jeromekelleher) [inline docs/genetic.md:391]: Don't need to ``print`` here I think, you can just do ``np.var(genetic_result.genetic["genetic_value"])`` like in a Jupyter notebook

**Review by jeromekelleher — COMMENTED** [review]
Looks good, but doesn't really address the point about how we might normalise more generally.

**Comment by daikitag** [comment]
I guess this pull request can be closed now. If users are interested in scaling, they can scale the simulated traits by using `normalise_genetic_value` or `normalise_phenotype` function, and they are well documented in the tstrait documentation. The simulated phenotypes will not explode after normalized, and the statistical properties of the traits are not distorted by using these normalizing functions (this has been thoroughly examined through validation against external simulators).

**Pull request closed without merging**